packages/blocks/src/store/selectors.js

```diff
@@ -75,6 +75,20 @@ export function getBlockStyles( state, name ) {
 	return state.blockStyles[ name ];
 }
 
+/**
+ * Returns default block style by block name.
+ *
+ * @param {Object} state Data state.
+ * @param {string} name  Block type name.
+ *
+ * @return {Array?} Block Styles.
+ */
+export function getDefaultBlockStyle( state, name ) {
+	return getBlockStyles( state, name ).filter(
```

Comment:
`find` from Lodash might be safer because of `filter` on `undefined` (`@return {Array?} Block Styles.`) will throw an error.